### PR TITLE
Fix tagging issues in merge automation

### PR DESCRIPTION
**Pull request by srfraser**

### CODE REVIEW

File: treescript/src/treescript/merges.py [hunk 1]
```diff
@@ -113,14 +113,17 @@ def touch_clobber_file(repo_path):
 
     await run_hg_command(config, "pull", "https://hg.mozilla.org/mozilla-unified", repo_path=repo_path)
 
-    await run_hg_command(config, "up", "-C", from_branch, repo_path=repo_path)
+    # Used if end_tag is set.
+    await run_hg_command(config, "up", "-C", to_branch, repo_path=repo_path)
+    to_fx_major_version = get_version("browser/config/version.txt", repo_path).major_number
```
Comment (JohanLorenzo) [inline treescript/src/treescript/merges.py:118]: Nice to see mozilla-version being used \o/

File: treescript/src/treescript/merges.py [hunk 2]
```diff
@@ -140,11 +143,9 @@ def touch_clobber_file(repo_path):
 
     end_tag = merge_config.get("end_tag")  # tag the end of the to repo
     if end_tag:
-        to_fx_major_version = get_version("browser/config/version.txt", repo_path).major_number
-        base_to_rev = await get_revision(config, repo_path, branch=to_branch)
         end_tag = end_tag.format(major_version=to_fx_major_version)
-        tag_message = f"No bug - tagging {os.path.basename(repo_path)} with {end_tag} a=release DONTBUILD CLOSED TREE"
-        await run_hg_command(config, "tag", "-m", f'"{tag_message}"', "-r", base_to_rev, "-f", end_tag, repo_path=repo_path)
+        tag_message = f"No bug - tagging {end_tag} with {end_tag} a=release DONTBUILD CLOSED TREE"
```
Comment (jcristau) [inline treescript/src/treescript/merges.py]: this should be "tagging {base_to_rev} with ..."

Comment (JohanLorenzo) [inline treescript/src/treescript/merges.py]: Nit: 

```suggestion
        tag_message = f"No bug - tagging {base_to_rev} with {end_tag} a=release DONTBUILD CLOSED TREE"
```

**Review by jcristau — CHANGES_REQUESTED** [review]

**Review by JohanLorenzo — APPROVED** [review]
LGTM modulo the nit!

As a followup, let's write a test suite to ensure we don't break commit messages anymore 🙂

**Review by jcristau — APPROVED** [review]
thanks for the fixes!

**Pull request merged**